### PR TITLE
added Mapco 

### PR DESCRIPTION
**Pull request by aayush920**

### CODE REVIEW

**Comment by arch0345** [comment]
Closes #5149

File: data/brands/amenity/fuel.json [hunk 1]
```diff
@@ -49,6 +49,21 @@
...
+      "locationSet": {
+        "include": ["001"],
+        "exclude": ["pl"]
+      },
+      "matchNames": ["mapco", "MAPCO"],
+      "tags": {
+        "amenity": "fuel",
+        "brand": "Mapco",
+        "brand:wikidata": "Q107589462",
+        "brand:wikipedia": "en:Mapco",
```
Comment (arch0345) [inline data/brands/amenity/fuel.json:63]: There doesn't seem to be an article for Mapco in the English Wikipedia, only Wikidata tags are required. You could make a Wikipedia article, but note that it's more difficult than creating a Wikidata page.

File: data/brands/amenity/fuel.json [hunk 2]
```diff
@@ -49,6 +49,21 @@
         "name": "7-Eleven"
       }
     },
+    {
+      "displayName": "Mapco",
+      "locationSet": {
+        "include": ["001"],
```
Comment (arch0345) [inline data/brands/amenity/fuel.json:55]: From my understanding, Mapco only has locations in the US. You should change the locationSet to include 'us' and remove the exclude argument.

**Review by arch0345 — CHANGES_REQUESTED** [review]
Thanks for your first addition to the name suggestion index! You should fix the locationSet to include 'us' rather than '001' (worldwide) and remove the exclude argument and also remove the brand:wikipedia tag. There doesn't seem to be a Wikipedia article for this brand yet and this is not a requirement for a brand to be added to the NSI.

**Comment by aayush920** [comment]
Thank you for telling. I have done the needfull.

**Review by arch0345 — APPROVED** [review]

**Comment by arch0345** [comment]
Thanks @aayush920!

**Pull request merged**